src/components/Logos/Logos.jsx

```diff
@@ -46,10 +46,10 @@ const Logos = ({ scrollRef }) => (
       style={{
         fontSize: "5rem",
         justifyContent: "center",
-        color: COLORS.colorHackRed,
+        color: COLORS.buttonColorText,
```

Comment:
nit: don't use general color (yellow, blue, etc.) or define new color (so colors are reusable between applications)

Reply:
> 

I will remove this file from this current commit as if i rename the variable it will cause issues in the dev branch when we merge and performing the merge into this branch will break PR #4.

Rather I suggest we merge now and I push an updated commit removing the usage of specific applications?